approaching-a-problem/pseudocode-and-logic.md

```diff
@@ -300,13 +290,13 @@ Which of the following is an accurate description of the process in Examples 1 a
 ##### !end-question
 ##### !options
 
-* The developer started with equal parts of readable pseudocode and working Python code
+* The developer started by writing out subproblems in a numbered list, then used that list to write working Python code
```

Comment:
Is this option the answer? Because it reads as correct to me. Could you clarify what I'm missing, or could we chaneg this option to something more "definitely not the correct answer"?

Reply:
Its missing anything about pseudocode, so it felt pretty clearly wrong to me, but please let me know what you'd suggest I change to make it more clearly wrong.